tests/etl/test_buildapi.py

```diff
@@ -165,6 +165,54 @@ def test_ingest_builds4h_jobs(jm, initial_data,
     assert len(stored_obj) == 20
 
 
+def test_ingest_running_to_complete_job(jm, initial_data,
```

Comment:
this also would have caught the bug we hit with not transitioning from running to complete.